app/views/pages/timed_retention.html.haml

```diff
@@ -1,29 +1,29 @@
-= render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
-
-- stale_weeks    = Settings.timed_transition_stale_weeks
-- archived_weeks = Settings.timed_transition_pending_weeks
+= render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
 
 %section.timed-retention
   %p
-    Starting from 19th September 2016, claims will only be retained on the system for a limited period, and not indefinitely.
+    = t('.paragraph_1')
+
   %p
-    Claims that are in one of the following states for #{stale_weeks} weeks, AND have had no messages added to the claim for #{stale_weeks} weeks will have their status changed to archived:
+    = t('.paragraph_2', stale_weeks: "#{Settings.timed_transition_stale_weeks}")
 
   %ul.timed-retention-states
     %li
-      draft
+      = t('draft').capitalize
     %li
-      authorised
+      = t('authorised').capitalize
     %li
-      part-authorised
+      = t('part_authorised').capitalize
     %li
-      refused
+      = t('refused').capitalize
```

Comment:
I think this could be wrapped up into an array that builds the list dynamically something like:

```ruby
states = ['draft', 'authorised', 'part_authourised', 'refused', 'rejected']
states.each_with_index do |state, index|
  index.eql?(states.length - 1) ? '%li' : '%li.last_element'
  = t(state).capitalize 
end
```
Though this is not a big thing, and definitely not a blocker. More of something we could look into to further DRY out the views. I have seen similar hardcoded lists throughout the application, some form of view helper would be a better way of managing the repetition.